pkg/registry/registrydeployer.go

```diff
@@ -77,26 +76,6 @@ func (r *RegistryDeployer) CreateRegistryResources(key types.NamespacedName, dis
 	return nil
 }
 
-// EnsurePackagesInStatus makes sure that the csc's status.PackageRepositioryVersions
-// field is updated at the end of the configuring phase if successful. It iterates
-// over the list of packages and creates a new map of PackageName:Version for each
-// package in the spec.
-func (r *RegistryDeployer) EnsurePackagesInStatus(csc *marketplace.CatalogSourceConfig) {
```

Comment:
Not sure why this is being removed in this commit

Reply:
responded here https://github.com/operator-framework/operator-marketplace/pull/198#discussion_r289464571